### ISSUE TITLE
[bug] rightmost table cells containing space only rendered without right side borders

### ISSUE THREAD

**Issue by revelt** [shd101wyy/markdown-preview-enhanced#407]
Hi all,

Check this code:

    aaa  | Description
    -----|-------------------
    {    |  
    text | zzzz
    }    |  

The cells that contain only spaces are rendered without right side borders:

![screen shot 2017-05-15 at 23 04 07](https://cloud.githubusercontent.com/assets/8344688/26081492/e67b5748-39c2-11e7-83f1-0a108128431c.png)

It's fine on GitHub later, when such code is put into readme.

Basically, let's check right border rendering when cell contains no letters (only spaces or nothing).

Tested on: MacOS Sierra, Atom 1.16.0 and markdown-preview-enhanced v0.12.3

**Comment by shd101wyy** [comment]
Will be fixed next week.
Thanks :)

**Comment by shd101wyy** [comment]
@revelt It seems that multiline is not supported in markdown table, you need to use `<br>` tag for line break:  

```markdown
aaa  | Description
-----|-------------------
{ <br> text <br> }    |   zzz
```
